src/web/page/templates.rs

```diff
@@ -250,6 +254,43 @@ fn dedent(value: &Value, _args: &HashMap<String, Value>) -> TeraResult<Value> {
     ))
 }
 
+enum IconType {
+    Strong,
+    Regular,
+    Brand,
+}
+
+impl fmt::Display for IconType {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let icon = match self {
+            Self::Strong => "fas",
+            Self::Regular => "far",
+            Self::Brand => "fab",
+        };
+
+        f.write_str(icon)
+    }
+}
+
+impl tera::Filter for IconType {
+    fn filter(&self, value: &Value, args: &HashMap<String, Value>) -> TeraResult<Value> {
+        let icon = format!(
+            r#"<span class="{} fa-{}" aria-hidden="true" {}></span>"#,
+            self,
+            value.as_str().expect("Icons only take strings"),
+            args.get("extra")
+                .and_then(Value::as_str)
+                .unwrap_or_default(),
```

Comment:
We need to be careful here, as we might get XSS issues if we ever allow users to choose a dynamic icon. A possible approach would be:

```rust
args.entry("class".to_string())
    .or_insert_with(String::new())
    .push_str(&format!(
		"{} fa-{}",
		self,
		value.as_str().expect("Icons only take strings"),
	));
args.entry("aria-hidden".to_string()).or_insert_with(|| "true".to_string());

let tag = "<span".to_string();
for (key, value) in args.iter() {
    tag.push(' ');
    tag.push_str(&tera::escape_html(key));
    tag.push_str("=\"");
    tag.push_str(&tera::escape_html(value));
    tag.push('"');
}
tag.push_str("></span>");
Ok(Value::String(tag));
```

This way, it's possible to turn:

```
{{ "copy" | far(extra='id="clipboard" aria-label="Copy crate name and version information"') }}
```

...into:

```
{{ "copy" | far(id="clipboard" aria-label="Copy crate name and version information") }}
```

...and we also get HTML escaping too!